### ISSUE TITLE
Dashboard Preview - Crawler auth setup failed - role not found

### ISSUE THREAD

**Issue by gannaramu** [grafana/grafana#51948]
**What happened**: Unable to get dashboard Previews to work in Grafana docker image (grafana/grafana-enterprise:9.0.2) 

**What you expected to happen**: 

**How to reproduce it (as minimally and precisely as possible)**:  
Dockerfile:
```
FROM grafana/grafana-enterprise:9.0.2
COPY grafana.ini /etc/grafana/grafana.ini
```
grafana.ini:
```
[rendering]
# Options to configure a remote HTTP image rendering service, e.g. using https://github.com/grafana/grafana-image-renderer.
# URL to a remote HTTP image renderer service, e.g. http://localhost:8081/render, will enable Grafana to render panels and dashboards to PNG-images using HTTP requests to an external service.
server_url = http://grafana-image-renderer:8081/render
# If the remote HTTP image renderer service runs on a different server than the Grafana server you may have to configure this to a URL where Grafana is reachable, e.g. http://grafana.domain/.
callback_url = https://<hostname>.com/grafana/
rendering_ignore_https_errors = true

[feature_toggles]
# enable features, separated by spaces
enable = dashboardPreviews serviceAccounts
 
[server]
# Protocol (http, https, h2, socket)
protocol = http

# The ip address to bind to, empty will bind to all interfaces
;http_addr =

# The http port  to use
;http_port = 3000

# The public facing domain name used to access grafana from a browser
domain = <hostname>.com

# Redirect to correct domain if host header does not match domain
# Prevents DNS rebinding attacks
;enforce_domain = false

# The full public facing url you use in browser, used for redirects and emails
# If you use reverse proxy and sub path specify full url (with sub path)
root_url = %(protocol)s://%(domain)s/grafana/

# Serve Grafana from subpath specified in `root_url` setting. By default it is set to `false` for compatibility reasons.
serve_from_sub_path = true
```

**Anything else we need to know?**:  
logger=previews_service t=2022-07-08T04:32:48.28109841Z level=error msg="Crawler auth setup failed" err="role not found" crawlerAuthSetupTime=1.866039ms

**Environment**:
- Grafana version: v9.0.2 (0641b5d0c)
- Data source type & version: MySQL
- OS Grafana is installed on: Kubernetes
- User OS & Browser: 
- Grafana plugins:
- Others:

**Comment by zuchka** [comment]
have you configured any special auth in the grafana.ini?

also try increasing [the verbosity of the Grafana server logs](https://grafana.com/docs/grafana/latest/administration/configuration/#level) to `debug` and note any errors. For printing to console, [set the console logs to `debug`](https://grafana.com/docs/grafana/latest/administration/configuration/#level-1) as well.

**Comment by gannaramu** [comment]
no auth configured.
even with debug level it would only print out 
```logger=previews_service t=2022-07-15T22:17:51.624544123Z level=error msg="Crawler auth setup failed" err="role not found" crawlerAuthSetupTime=210.912µs```

**Comment by ArturWierzbicki** [comment]
hey @gannaramu, thank you for reporting this issue! Is your instance licensed? 

I was able to reproduce this issue only with a fresh, unlicensed Enterprise Grafana. The fix is already merged and scheduled to be released in 9.0.5

**Comment by gannaramu** [comment]
No my instance is not licensed. Thanks for the fix!
